start.py

```diff
@@ -14,11 +14,11 @@
 import sys
 import time
 
-sys.path.append('./test')
+sys.path.append("./test")
 import startservers
 
 if not startservers.install(race_detection=False):
-    raise(Exception("failed to build"))
+    raise (Exception("failed to build"))
```

Comment:
nit: remove the wrapping parens too, here and a few other places (`grep "raise ("`)

Reply:
Ooof, this one is in a ton of places.  Fixed!